manifests/params.pp

```diff
@@ -16,13 +16,15 @@
   $manage_dbserver           = true
 
   # The remaining database settings are not used for an embedded database
-  $database_host          = 'localhost'
-  $database_port          = '5432'
-  $database_name          = 'puppetdb'
-  $database_username      = 'puppetdb'
-  $database_password      = 'puppetdb'
-  $database_ssl           = false
-  $database_validate      = true
+  $database_host      = 'localhost'
+  $database_port      = '5432'
+  $database_name      = 'puppetdb'
+  $database_username  = 'puppetdb'
+  $database_password  = 'puppetdb'
+  $database_ssl       = false
+  $database_validate  = true
+  $postgres_version   = '9.4'
```

Comment:
I don't mind the default being the latest, I can see this being another annoying place to constantly update however :-). Kind of wish there was a way to just say 'give me the latest always' instead. But hohum, yes 9.4 is fine.